Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -165,4 +165,10 @@ enum TrackingEvent: String, Trackable {
     case addDelegateKeySkipped                      = "user_skip_add_delegate"
     case addDelegateKeySuccess                      = "user_success_add_delegate"
     case addDelegateKeyFailed                       = "user_failed_add_delegate"
+
+    // MARK: delete  delegate key
```

Comment:
👍🏻 
Could you please also update our tracking document here? https://docs.google.com/spreadsheets/d/1mj9iQIhpM-Pak7lQMb1dmxhQn1QdJoJyhbc730dZNzA/edit#gid=1949479240